docs/console/company-configuration/extensions.md

```diff
@@ -68,35 +71,110 @@ On this page, you can:
 
 ## Extension Detail
```

Comment:
I wonder if it is ok to remove the sections _Extension Info_, _Extension Location_ and _Extension Visibility_, given the little amount of information included in these paragraphs, and group them in just one section.

The UI however includes a link to each of the three cards related to the aforementioned sections of the documentation. These card are in the same page.

I wonder if it is better to maintain the previous approach (multiple paragraph) in order to have a more consistent UI, or to keep going with the approach suggested in this PR.